src/constants/indicators.js

```diff
@@ -179,59 +179,89 @@ export default {
     contextual: false,
     source: 'Education attainment - Completed Year 12, % of population',
   },
+  educationalAttainmentCertificate: {
+    name: 'Educational attainment - Certificate',
+    shortDescription: 'TODO',
+    longDescription: 'TODO',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0%',
+    contextual: false,
+    source: 'Education attainment - Completed Certificate III or IV, % of population',
+  },
+  educationalAttainmentTertiary: {
+    name: 'Educational attainment - Tertiary',
+    shortDescription: 'TODO',
+    longDescription: 'TODO',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0%',
+    contextual: false,
+    source: 'Education attainment - Completed Bachelor Degree or Higher, % of population',
+  },
   dispersion: {
     name: 'Local government dispersion',
-    shortDescription: 'Local government dispersion',
-    longDescription: 'Local government dispersion',
+    shortDescription: 'LGAs per 100,000 persons',
+    longDescription: 'Number of Local Government Areas per 100 000 persons',
```

Comment:
Should "Local Government Areas" be lower case?

Reply:
I believe it's an Official Term that Means Something Specific.

Reply:
heh